### PR TITLE
added amazon and google

### PR DESCRIPTION
**Pull request by sbromberger**

### CODE REVIEW

File: src/SNAPDatasets.jl [hunk 1]
```diff
@@ -5,6 +5,7 @@ using LightGraphs
 const SNAPDIR = joinpath(@__DIR__,"..","data")
 
 snap_graphs = Dict{Symbol, String}(
+    :amazon0302         => "amazon0302/amazon0302.jlg",
 	:arxiv_cit_hep_th	=> "arxiv_cit/arxiv-cit-hep-th.jlg",
```
Comment (simonschoelly) [inline src/SNAPDatasets.jl]:
```suggestion
        :amazon0302         => "amazon0302/amazon0302.jlg",
	:arxiv_cit_hep_th	=> "arxiv_cit/arxiv-cit-hep-th.jlg",
```

Comment (simonschoelly) [reply src/SNAPDatasets.jl]: I think that should fix the spacing.

**Comment by simonschoelly** [comment]
Should we have some tests for this? Otherwise we can merge it, I guess.

**Comment by sbromberger** [comment]
I don't think we need tests. There's a single function.

**Comment by sbromberger** [comment]
That test might take a while. :) But in any case, I think we're ok here.

**Pull request merged**